pkg/asset/imagebased/image/registriesconf.go

```diff
@@ -73,6 +74,7 @@ func (i *RegistriesConf) generateRegistriesConf(imageDigestSources []types.Image
 		registry := sysregistriesv2.Registry{}
 		registry.Endpoint.Location = group.Source
 		registry.MirrorByDigestOnly = true
+		registry.Blocked = group.SourcePolicy == configv1.NeverContactSource
```

Comment:
This only applies to image-based install, right? We need to add this in the bootstrap common for core install:

https://github.com/openshift/installer/blob/8ecd9449b0eddf0c13a41ce41660bd52493b0fd0/pkg/asset/ignition/bootstrap/common.go#L308-L314